pkg/volume/csi/csi_plugin.go

```diff
@@ -68,10 +66,9 @@ const (
 var deprecatedSocketDirVersions = []string{"0.1.0", "0.2.0", "0.3.0", "0.4.0"}
 
 type csiPlugin struct {
-	host              volume.VolumeHost
-	blockEnabled      bool
-	csiDriverLister   csilister.CSIDriverLister
-	csiDriverInformer csiinformer.CSIDriverInformer
+	host            volume.VolumeHost
+	blockEnabled    bool
+	csiDriverLister storagelisters.CSIDriverLister
```

Comment:
Do we need to store csiDriverLister here, or can we just get it from the volumehost when we need it?

Reply:
It's more convenient to keep it here, but maybe not required.

Reply:
Removed csiDriverLister from csiPlugin struct.